src/content/docs/en/core-concepts/routing.mdx

```diff
@@ -239,6 +239,8 @@ Astro needs to know which route should be used to build the page. To do so, it s
 - Rest parameters have the lowest priority
 - Ties are resolved alphabetically
 
+sMoreover, endpoints always take precedence over pages. Also, if you have dynamic routes (using rest parameters or named parameters) that are prerendered, they will take precedence too over dynamic routes that aren't.
+
```

Comment:
```suggestion
- Pre-rendered dynamic routes take precedence over static dynamic routes
- Rest parameters have the lowest priority
- Endpoints always take precedence over pages
- Ties are resolved alphabetically
```
What do you think about just adding to our list, like this? (view it with the points in the lines just above)

Reply:
That works for me, thanks for suggesting that!
There's just a minor change to do, in the first bullet point in “over static dynamic routes” it should be “**server**” instead of “**static**”.

Reply:
Perfect, thanks!